### PR TITLE
5211 - Fixed Bug Issue in Mobile Selection Range Rendering

### LINKED ISSUE #5211: Datepicker: Selection of range not rendered properly on mobile
**Describe the bug**
Datepicker: Selection of range not rendered properly on mobile

**To Reproduce**

1. Using your mobile device/Chrome's Toggle devic toolbar, go to https://main-enterprise.demo.design.infor.com/components/datepicker/example-range.html?theme=new&mode=light&colors=0066D4
2. Click on any date picker icon that requires range
3. Click on any range 

**Expected behavior**
Selection of range should be rendered properly on mobile

**Version**
v4.52.0-dev

**Screenshots**
iPhone X is used in the screenshot
Actual Result
Actual Result - Range

Expected result
Expected Result - Range

**Platform**
 - All mobiles devices

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by RiyaTayal20**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR fixes the selection range rendering in the mobile view.

**Related github/jira issue (required)**:
Closes #5211 

**Steps necessary to review your pull request (required)**:

- Pull this branch down, then build and run the app
- Go to http://localhost:4000/components/datepicker/example-range.html
- Open up mobile view
- Check if selection range rendering looks like the following: 
Screen Shot 2021-06-03 at 10 19 50 AM

**Included in this Pull Request**:
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Looks good but requesting two small changes

1. Can you add a change log entry to https://github.com/infor-design/enterprise/blob/main/docs/CHANGELOG.md (for all issues in general)
2. I noticed if you zoom to 75% there is a tiny white border... This seems to be easily fixable by adding margin-left: -1px

Screen Shot 2021-06-03 at 1 10 07 PM
Screen Shot 2021-06-03 at 1 13 01 PM

**Review by EdwardCoyle — APPROVED** [review]
@RiyaTayal20 looks great! 👍

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 ### v4.52.0 Markup Changes
 
 - `[Datagrid]` When fixing bugs in datagrid hover states we removed the use of `is-focused` on table `td` elements. ([#5091](https://github.com/infor-design/enterprise/issues/5091))
+- `[Datepicker]` Fixed a bug where the selection range was not being properly rendered in mobile. ([#5211](https://github.com/infor-design/enterprise/issues/5211))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:12]: Just please move this down to 4.52 Fixes section as you actually didn't change markup (this is more like noting breaking markup changes) Than looks great 👍🏻

Comment (RiyaTayal20) [reply docs/CHANGELOG.md:12]: I'll make sure to do that now

**Review by tmcconechy — APPROVED** [review]

**Review by deep7102 — APPROVED** [review]

**Pull request merged**